src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -309,6 +309,35 @@
         "field_namespace": "eiaaeo",
         "etl_group": "eiaaeo",
     },
+    "core_eiaaeo__yearly_projected_delivered_energy": {
+        "description": (
+            "Projected energy delivered to residential, commercial, industrial, and "
+            "transportation sectors. Industrial includes energy for "
+            "combined-heat-and-power plants that have a non-regulatory status and "
+            "small on-site generating systems."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "region_name_eiaaeo",
+                "region_type_eiaaeo",
+                "model_case_eiaaeo",
```

Comment:
I dug into this column a little bit, and everything seems reasonable except for the `aeo2022` value -- do you know what that represents? It would help to explain that either in the definition for this column or as a comment next to the value in the `MODEL_CASES_EIAAEO` list in `enums.py`